README.md

```diff
@@ -1,2 +1,2 @@
 # application-api
-App Studio Application API
+App Studio Application API. 
```

Comment:
This change probably doesn't need to be part of this PR.

Reply:
removed